README.md

```diff
@@ -16,11 +16,15 @@ gem "lita-giphy"
 ### Required attributes
 * `api_key` - Your [Giphy API](https://github.com/giphy/GiphyAPI) key. You can either email the devs for a personal API key, or you can use the default public beta key-- `dc6zaTOxFJmzC`.
 
+### Optional attributes
+* `rating` - allows you to set the rating of the gifs that are found
+
 ### Example
 
 ``` ruby
 Lita.configure do |config|
   config.handlers.giphy.api_key = "dc6zaTOxFJmzC"
+  config.handlers.giphy.rating
```

Comment:
This should probably be set to something, right? `config.handlers.giphy.rating = 'g'`?